src/node.rs

```diff
@@ -550,6 +550,12 @@ impl<'a> NodeProperty<'a> {
         core::str::from_utf8(self.value).map(|s| s.trim_end_matches('\0')).ok()
     }
 
+    /// Attempts to parse the property value as a list of [`&str`].
+    pub fn as_str_list(self) -> Option<impl Iterator<Item = &'a str> + 'a> {
```

Comment:
A better method name would be ideal, something akin to `iter_strs` perhaps? (the [API Naming Guidelines](https://rust-lang.github.io/api-guidelines/naming.html#ad-hoc-conversions-follow-as_-to_-into_-conventions-c-conv) have a specific meaning for methods beginning with `as_`, so I'd like to try and keep things relatively consistent with that)

Reply:
Sounds good to me, I'll change it.